index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <header>
+      <a href="#" class="logo">My Logo</a>
+      <nav class="navbar">
+        <img src="image/Union.png" alt="" />
+      </nav>
+    </header>
+    <section class="section-1">
+      <div class="sectionbackground">
+        <div class="desc">
+          <h1 class="marianne">I’m Mariane Glad to see you!</h1>
```

Comment:
- [ optional] kindly add the font weight here to make it more like the figma and more recognizable.